tests/ignite/contrib/engines/test_common.py

```diff
@@ -376,14 +376,15 @@ def test_setup_plx_logging():
 
     os.environ["POLYAXON_NO_OP"] = "1"
 
-    _test_setup_logging(
-        setup_logging_fn=setup_plx_logging,
-        kwargs_dict={},
-        output_handler_cls=handlers.polyaxon_logger.OutputHandler,
-        opt_params_handler_cls=handlers.polyaxon_logger.OptimizerParamsHandler,
-        with_eval=False,
-        with_optim=False,
-    )
+    with pytest.warns(DeprecationWarning):
```

Comment:
Please, specify warn part of message to match:
```python
with pytest.warns(DeprecationWarning, match=r"bla-bla-bla"):
    # ...
```

Reply:
Done